fineract-provider/src/main/java/org/apache/fineract/portfolio/note/domain/Note.java

```diff
@@ -37,23 +42,27 @@
 import org.apache.fineract.portfolio.shareaccounts.domain.ShareAccount;
 
 @Entity
+@Getter
+@NoArgsConstructor
```

Comment:
No-one should call the NoArgsConstructor, JPA is fine with protected access level, so could you please change it to be protected.

Reply:
Noted.